game-core/src/main/java/games/strategy/engine/framework/ui/DefaultGameChooserEntry.java

```diff
@@ -0,0 +1,132 @@
+package games.strategy.engine.framework.ui;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.logging.Level;
+
+import games.strategy.engine.data.EngineVersionException;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GameParseException;
+import games.strategy.engine.data.GameParser;
+import games.strategy.triplea.Constants;
+import games.strategy.util.UrlStreams;
+import lombok.extern.java.Log;
+
+@Log
+final class DefaultGameChooserEntry implements GameChooserEntry {
```

Comment:
This class is identical to the original `GameChooserEntry` class except for two things.  The first is that the new class is now `final`.